tests/console/ncurses.pm

```diff
@@ -24,7 +24,10 @@ use utils qw(clear_console zypper_call);
 sub run {
     select_console 'root-console';
     zypper_call 'in dialog';
-    script_run 'dialog --yesno "test for boo#1054448" 3 20';
+    # Try to draw a bold red line
+    script_run('echo "$(tput smacs;tput setaf 1;tput bold)lqqqqqqqqqqqqqqk$(tput rmacs;tput sgr0)"');
+    # Try a simple yes/no dialog
+    script_run 'dialog --yesno "test for boo#1054448" 3 20 | tee boo_1054448.tee" 3 20';
```

Comment:
As a temporary measure this is ok but for the future I would prefer to have more simple test statements first and only then more complicated steps. So for example I would do

```perl
    enter_cmd 'dialog --yesno "test for boo#1054448" 3 20';
    assert_screen([qw(ncurses-simple-dialog ncurses-simple-dialog-broken-boo1183234)]);
    die "boo#1183234: Console misconfigured, ncurses UI broken!\n"
      if match_has_tag 'ncurses-simple-dialog-broken-boo1183234';
    # Try to draw a bold red line
    script_run('echo "$(tput smacs;tput setaf 1;tput bold)lqqqqqqqqqqqqqqk$(tput rmacs;tput sgr0)"');
    # Try a simple yes/no dialog
    script_run 'dialog --yesno "test for boo#1054448" 3 20 | tee boo_1054448.tee" 3 20';
```

Would this work for you?

Reply:
I would say the red line test is simpler, that's why I put it first.
Then, I just added the `| tee` part to get hidden output (which is uploaded only on failure).

Reply:
Well, I saw that you added `… | tee` this is why I repeated the last call of dialog to have the most simple first. Do you mean to have "tput" tested before "dialog" as something more low-level? That would make sense and that would be of course ok but is this still related to "ncurses" then?

Reply:
I see no point to run it without `|tee`.
The `tput` part is indeed used to check lower level bits.

Yes, I need this PR to be merge, because I do not have the required data collected, and this could be useful for other products as well.